components/ScrollButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react";
+
+import CustomImage from "./CustomImage";
+
+export default function Navbar() {
```

Comment:
Looks like this should be named `ScrollButton`? Although tbh I make this mistake everywhere and the name doesn't actually matter :sweat_smile:

Reply:
LMAO i copied and pasted from another file and forgot to change...